plugins/usbdmx/DMXCProjectsNodleU1.cpp

```diff
@@ -452,32 +481,61 @@ void DMXCProjectsNodleU1AsyncUsbSender::PostTransferHook() {
 }
 
 bool DMXCProjectsNodleU1AsyncUsbSender::ContinueTransfer() {
-  unsigned int length = 32;
+  if (!m_portId) {
+    unsigned int length = 32;
```

Comment:
the "legacy" protocol (sending 32 byte long blocks) is used for universe "0", even if the more advanced protocol could be used instead